### PR TITLE
Loosen Codeowners restrictions

### PR DESCRIPTION
**Pull request by iHiD**
PRs require one CodeOwner reviewer to approve. This loosens the existing requirements somewhat by allowing Erik/Karlo to be autonomous on reviewing/merging any JS PRs, and removes codeowner restrictions from other non-critical files.

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1 +1,18 @@
...
+*.rb      @iHiD
+*.ru      @iHiD
+Gemfile*  @iHiD
+config/   @iHiD
+db/       @iHiD
+lib/      @iHiD
+
+# Docker and deployment files
+docker/   @iHiD
+Procfile* @iHiD
```
Comment (ErikSchierboom) [inline .github/CODEOWNERS]: Maybe also add the `.github` directory?

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -1 +1,18 @@
...
+# Ruby files
+*.rb      @iHiD
+*.ru      @iHiD
+Gemfile*  @iHiD
+config/   @iHiD
+db/       @iHiD
+lib/      @iHiD
+
+# Docker and deployment files
+docker/   @iHiD
```
Comment (ErikSchierboom) [inline .github/CODEOWNERS]: Do you also want to add the `dev.Dockerfile` here?

File: .github/CODEOWNERS [hunk 3]
```diff
@@ -1 +1,18 @@
-* @iHiD
+# JS files
+*.js      @iHiD @kntsoriano @erikschierboom
+
+# Ruby files
```
Comment (ErikSchierboom) [inline .github/CODEOWNERS:4]: Is the `Rakefile` usual to require review?

**Review by ErikSchierboom — APPROVED** [review]

File: .github/CODEOWNERS [hunk 4]
```diff
@@ -1 +1,18 @@
...
+*.rb      @iHiD
+*.ru      @iHiD
+Gemfile*  @iHiD
+config/   @iHiD
+db/       @iHiD
+lib/      @iHiD
```
Comment (iHiD) [inline .github/CODEOWNERS:10]:
```suggestion
lib/      @iHiD
Rakefile  @iHiD
```

File: .github/CODEOWNERS (hunk 3 above)
Comment (iHiD) [reply .github/CODEOWNERS:4]: Good shout. Thanks.

File: .github/CODEOWNERS (hunk 2 above)
Comment (iHiD) [reply .github/CODEOWNERS]:
```suggestion
docker/          @iHiD
dev.Dockerfile   @erikschierboom @iHiD @kntsoriano
```

File: .github/CODEOWNERS [hunk 5]
```diff
@@ -1 +1,18 @@
-* @iHiD
+# JS files
+*.js      @iHiD @kntsoriano @erikschierboom
```
Comment (iHiD) [inline .github/CODEOWNERS]:
```suggestion
*.js      @erikschierboom @iHiD @kntsoriano 
```

Comment (iHiD) [reply .github/CODEOWNERS]: (Keep them alphabetical)

File: .github/CODEOWNERS (hunk 1 above)
Comment (iHiD) [reply .github/CODEOWNERS]:
```suggestion
Procfile*        @iHiD
.github          @iHiD
```

**Review by kntsoriano — APPROVED** [review]
Looks good! I'm unsure if we should loosen restrictions on `test/system/components` as well?

**Comment by iHiD** [comment]
I'm happy to keep reviewing those for now. Thanks! Merging 🙂

**Pull request merged**